packages/cozy-client/types/associations/Association.d.ts

```diff
@@ -77,7 +77,7 @@ declare class Association {
      *
      * @returns {QueryDefinition | QueryState}
      */
-    static query(): QueryDefinition | QueryState;
+    static query(document: any, client: any, assoc: any): QueryDefinition | QueryState;
```

Comment:
you can type them. 
document can be CozyClientDocument
client should be CozyClient 
it seems we have an `Association` type also

Reply:
cf https://github.com/cozy/cozy-client/pull/964#discussion_r652811977